kitsune/wiki/views.py

```diff
@@ -1656,24 +1657,39 @@ def recent_revisions(request):
     fragment = request.GET.pop("fragment", None)
     form = RevisionFilterForm(request.GET)
 
-    # We are going to ignore validation errors for the most part, but
-    # this is needed to call the functions that generate `cleaned_data`
-    # This helps in particular when bad user names are typed in.
+    # Validate form to populate `cleaned_data`
     form.is_valid()
 
     filters = {}
-    # If something has gone very wrong, `cleaned_data` won't be there.
     if hasattr(form, "cleaned_data"):
         if form.cleaned_data.get("locale"):
             filters.update(document__locale=form.cleaned_data["locale"])
-        if form.cleaned_data.get("users"):
-            filters.update(creator__in=form.cleaned_data["users"])
+
+        users = form.cleaned_data.get("users")
+
+        # If `users` exists, find matching User or Profile instances
+        if users:
```

Comment:
We could just update the code in the `MultiUsernameField` to do a search for both username and display name.

Reply:
See my other note.